packages/clickable/tests/clickable.test.tsx

```diff
@@ -1,90 +1,114 @@
   it("has role attribute", () => {
-    const wrapper = shallow(
+    const { getByRole } = render(
       <Clickable action={jest.fn()}>
         <span>onClick</span>
       </Clickable>
     );
-    expect(wrapper.prop("role")).toBe("button");
+
+    const buttonElement = getByRole("button");
+
+    expect(buttonElement).toBeInTheDocument();
   });
   it("has onClick function", () => {
     const action = jest.fn();
-    const wrapper = shallow(
+    const { getByRole } = render(
       <Clickable action={action}>
         <span>onClick</span>
       </Clickable>
     );
-    wrapper.simulate("click");
+
+    expect(action).not.toHaveBeenCalled();
+
+    const buttonElement = getByRole("button");
+
+    fireEvent.click(buttonElement);
     expect(action).toHaveBeenCalled();
   });
 
   it("has onKeyPress function and reacts on space", () => {
     const action = jest.fn();
-    const wrapper = shallow(
+    const { getByRole } = render(
       <Clickable action={action}>
         <span>onKeyPress</span>
       </Clickable>
     );
-    wrapper.simulate("keyPress", {
+
+    const buttonElement = getByRole("button");
+
+    fireEvent.keyPress(buttonElement, {
       key: " ",
-      keyCode: 32,
-      which: 32
+      code: "Space",
+      charCode: 32
     });
     expect(action).toHaveBeenCalled();
   });
 
   it("has onKeyPress function and reacts on Enter", () => {
     const action = jest.fn();
-    const wrapper = shallow(
+    const { getByRole } = render(
       <Clickable action={action}>
         <span>onKeyPress</span>
       </Clickable>
     );
-    wrapper.simulate("keyPress", {
+
+    const buttonElement = getByRole("button");
+
+    fireEvent.keyPress(buttonElement, {
       key: "Enter",
-      keyCode: 13,
-      which: 13
+      code: "Enter",
+      charCode: 13
     });
     expect(action).toHaveBeenCalled();
   });
   it("does not react on e keypress", () => {
     const action = jest.fn();
-    const wrapper = shallow(
+    const { getByRole } = render(
       <Clickable action={action}>
         <span>onKeyPress</span>
       </Clickable>
     );
-    wrapper.simulate("keyPress", {
+
+    const buttonElement = getByRole("button");
+
+    fireEvent.keyPress(buttonElement, {
       key: "e",
-      keyCode: 69,
-      which: 69
+      code: "KeyE",
+      charCode: 69
     });
     expect(action).not.toHaveBeenCalled();
   });
   describe("tabIndex", () => {
     it("default value", () => {
       const action = jest.fn();
-      const wrapper = shallow(
+      const { getByRole } = render(
         <Clickable action={action}>
-          <span>default tabIndex</span>
+          <span>onKeyPress</span>
         </Clickable>
       );
-      const tabIndex = wrapper.find("span").props().tabIndex || "";
-      expect(tabIndex.toString()).toEqual("-1");
+
+      const spanElement = getByRole("button");
+      const tabIndex = spanElement.getAttribute("tabIndex");
+
+      expect(tabIndex).toEqual("-1");
```

Comment:
shouldn't we test the user's experience here? 

i.e. we should test whether span element is focused or not on load, instead of checking it's tabIndex